app/services/foreman_ansible/proxy_load_balancer.rb

```diff
@@ -0,0 +1,36 @@
+module ForemanAnsible
+  class ProxyLoadBalancer
```

Comment:
That's what I meant in the initial comment https://github.com/theforeman/foreman_ansible/pull/40#issue-173284489